docs/language/contracts.mdx

```diff
@@ -387,6 +387,26 @@ let signer: AuthAccount = ...
 let contract = signer.contracts.get(name: "Test")
 ```
 
+### Borrowing a Deployed Contract
+
+A reference to a deployed contract contract can obtained using the `borrow` function:
```

Comment:
Maybe add a remark mentioning that this is the "dynamic" variant of the static imports. i.e: in a way they are "dynamic imports".
Also maybe some pros of it: the contract would be imported only if the execution path was hit (e.g: selective importing).